packages/main/src/MultiComboBox.js

```diff
@@ -585,6 +585,31 @@ class MultiComboBox extends UI5Element {
 		this[`_handle${event.key}`] && this[`_handle${event.key}`](event);
 	}
 
+	_handleShow(event) {
```

Comment:
1. If there is a value in the MultiComboBox and the picker is open via Arrow Up/Down + Alt or F4, the focus remains in the input field
2. When a token is selected and focused, opening the dropdown the focus goes to the corresponding suggestion but according to the specification, no matter whether the token is selected or not, when focused, on opening the dropdown the focus should go to the corresponding item. Currently, if just focused and not selected, to focus goes to the first suggestion which is not correct